lute/templates/book/create_new.html

```diff
@@ -95,6 +98,29 @@
 
     set_field_directions();
     $("#language_id").change(set_field_directions);
+
+    const textarea = document.getElementById("text");
+    const wordCountEl = document.getElementById("word-count");
+
+    textarea.addEventListener("input", getWordCountInfo);
+    textarea.addEventListener("click", getWordCountInfo);
+    textarea.addEventListener("selectionchange", getWordCountInfo);
+
+    function getWordCountInfo() {
+      let allWordsArray = [];
+      let wordsToCursorArray = [];
+
+      try {
+        allWordsArray = this.value.match(/\p{L}+/gu).filter((item) => item);
+        const selectionStart = (this.selectionEnd - this.selectionStart) > 0 ? this.selectionStart : 0;
+        const stringToCursor = this.value.slice(selectionStart, this.selectionEnd);
+        wordsToCursorArray = stringToCursor.match(/\p{L}+/gu).filter((item) => item);
```

Comment:
I don't think this will work with character-based languages or with Japanese.  This would require a server-side check, wouldn't it?